applications/ThresholdSignature.md

```diff
@@ -0,0 +1,300 @@
+- A Substrate based project called [ChainFlip](https://chainflip.io), which is a service that uses the FROST protocol to swap coins from one chain to another. 
+- A Substrate based project that implements a [Threshold ECDSA Distributed Key Generation Protocol](https://github.com/webb-tools/dkg-substrate). 
+
+Outside the Substrate / Polkadot / Kusama ecosystem, we have found the following:
+- ZCash Foundation has recently announced the first [stable release](https://zfnd.org/frost-reference-implementation-v1-0-0-stable-release/) of a FROST implementation, but has yet to integrate it in its ecosystem.
+- There have been attempts in the past to implement off-chain threshold Schnorr signing schemes in blockchain related projects, specifically by [Zengo](https://github.com/ZenGo-X/multi-party-schnorr). However, their implementation is based on an older protocol that is less efficient than the FROST protocol.
+- The same company has implemented a multi-signature scheme [Proof of Concept in Solana](https://github.com/ZenGo-X/solana-tss), but it is a n-of-n signature scheme, which is a special case of the more flexible and powerful t-of-n scheme.
+
+Other than that, we have not found any integration of a FROST based protocol in a relevant related ecosystem, which shows that a potential integration in the Substrate / Polkadot / Kusama ecosystem would provide a substantial advantage in this field relative to other competitors.
+
+### Team members
+
+- Rui Morais
+
+### Contact
+
+- **Contact Name:** Rui Morais
+- **Contact Email:** ruipedromorais11@gmail.com
+- **Github:** https://github.com/fiono11
+- **Linkedin:** https://www.linkedin.com/rui-morais
+
+### Team's experience
+
+- Recently submitted a PhD thesis in Computer Science entitled *"Contributions to Permissionless Decentralized Networks for Digital Currencies Based on Delegated Proof of Stake"* (awaiting for the defense).
+- Published the following papers:
+  - [Echidna: A New Consensus Algorithm for Efficient State Machine Replication](https://ieeexplore.ieee.org/document/10338927) (IEEE BCCA 2023)
+  - [Nero: A Deterministic Leaderless Consensus Algorithm for DAG-Based Cryptocurrencies](https://www.mdpi.com/1999-4893/16/1/38) (Algorithms 2022)
+  - [A tool for implementing privacy in Nano](https://ieeexplore.ieee.org/document/9126023) (IEEE DAPPS 2020)
+  - [Adamastor: a New Low Latency and Scalable Decentralized Anonymous Payment System](https://arxiv.org/abs/2011.14159) (Arxiv)
+- Has graduated the PBA Hong Kong cohort. 
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10,000 USD
+
+### Milestone 1 - Olaf Library Integration into [Schnorrkel repository](https://github.com/w3f/schnorrkel)
```

Comment:
$5,000 is too little money for this.  Increase it to $10,000.   We'll add a third reviewer.

Add me on element:  @jeff:web3.foundation and I'll help talk you through some of the schnorrkel code whenever you need help.  In particular, there are already DLEQ proofs which work with more than two point pairs, which matye useful. 

https://github.com/w3f/schnorrkel/blob/master/src/vrf.rs#L627

https://github.com/w3f/schnorrkel/blob/master/src/vrf.rs#L430

Reply:
> $5,000 is too little money for this. Increase it to $10,000. We'll add a third reviewer.
> 
> Add me on element: @jeff:web3.foundation and I'll help talk you through some of the schnorrkel code whenever you need help. In particular, there are already DLEQ proofs which work with more than two point pairs, which matye useful.
> 
> https://github.com/w3f/schnorrkel/blob/master/src/vrf.rs#L627
> 
> https://github.com/w3f/schnorrkel/blob/master/src/vrf.rs#L430

Thank you! I am really excited to do this project!